presto-main/src/main/java/com/facebook/presto/operator/scalar/annotations/SqlInvokedScalarFromAnnotationsParser.java

```diff
@@ -103,7 +104,7 @@ private SqlInvokedScalarFromAnnotationsParser() {}
         Set<Method> methods = findPublicStaticMethods(
                 clazz,
                 ImmutableSet.of(SqlInvokedScalarFunction.class, SqlType.class, SqlParameter.class, SqlParameters.class, Description.class),
-                ImmutableSet.of(ScalarFunction.class, ScalarOperator.class));
+                ImmutableSet.of(ScalarFunction.class, ScalarOperator.class, CodegenScalarFunction.class));
```

Comment:
This is needed because of the way SqlInvoked... parses annotations. If anything has @sqltype annotation, and not @scalarfunction annotation, it will assume its sqlinvoked

Reply:
I will do that in a different PR